user/prog.c

```diff
@@ -19,15 +20,44 @@ void marquee(const char *string, int offset) {
   textarea[TEXTAREA_SIZE - 1] = 0;
 }
 
+/* Temporarily required, because I don't want to mix printf with sbrk_test. */
+void assert(int cond) {
+  if (!cond)
```

Comment:
Please add dummy `sys_exit` implementation that will just set `td_state` to `TDS_INACTIVE` and switch to another thread. @psie will provide full implementation later.